e2e/tests/upgrades/upgrade_test.go

```diff
@@ -386,6 +391,147 @@ func (s *UpgradeTestSuite) TestV5ToV6ChainUpgrade() {
 	})
 }
 
+func (s *UpgradeTestSuite) TestV6ToV7ChainUpgrade() {
+	t := s.T()
+	testCfg := testconfig.FromEnv()
+
+	ctx := context.Background()
+	relayer, channelA := s.SetupChainsRelayerAndChannel(ctx)
+	chainA, chainB := s.GetChains()
+
+	createClientOptions := ibc.CreateClientOptions{
+		TrustingPeriod: time.Duration(time.Hour * 24 * 7).String(),
+	}
+
+	// create second tendermint client
+	s.SetupClients(ctx, relayer, createClientOptions)
+
+	status, err := s.Status(ctx, chainA, ibctesting.FirstClientID)
+	s.Require().NoError(err)
+	s.Require().Equal("Active", status)
+
+	status, err = s.Status(ctx, chainA, ibctesting.SecondClientID)
+	s.Require().NoError(err)
+	s.Require().Equal("Active", status)
+
+	var (
+		chainADenom         = chainA.Config().Denom
+		chainBIBCToken      = testsuite.GetIBCToken(chainADenom, channelA.Counterparty.PortID, channelA.Counterparty.ChannelID) // IBC token sent to chainB
+		soloMachineClientID = "06-solomachine-2"
```

Comment:
I left a comment later about a function to name client IDs, I think this is a good place to use it!